app/experimenter/experiments/constants/nimbus.py

```diff
@@ -226,6 +226,18 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_NO_ENTERPRISE_OR_PAST_VPN = NimbusTargetingConfig(
+    name="No enterprise or past VPN use",
+    slug="no_enterprise_or_past_vpn",
+    description="Exclude users who have used Mozilla VPN or who are enterprise users",
+    targeting=(
+        "{no_enterprise_users} && "
```

Comment:
This needs the little `f` at the start to be a format string and evaluate and substitute the `{no_enterprise_users}` part

```py
         f"{no_enterprise_users} && "
```

Reply:
Good catch; thanks!  I've pushed a fixed patch.